c7n/resources/elasticache.py

```diff
@@ -458,3 +458,16 @@ def _cluster_eligible_for_snapshot(cluster):
         cluster['Engine'] != 'memcached' and not
         TTYPE.match(cluster['CacheNodeType'])
     )
+
+
+@resources.register('replication-group')
```

Comment:
replication group is a little generic, this should probably be elasticache-group?

Reply:
Got it! Addressed